### ISSUE TITLE
Installation failed

### ISSUE THREAD

**Issue by InfiniteCod3** [ublue-os/bazzite#1016]
### Describe the bug

When installing the OS it fails with the command "ostree container image deploy --sysroot=/mnt/sysimage --image=/run/install/repo/bazzite/nvidia-stable --transport=oci --no-signature-verification" exited with the code 1;

### What did you expect to happen?

For it to successfully install

### Extra information or context

![1714030265130](https://github.com/ublue-os/bazzite/assets/91837651/6d13dc25-cadb-4d5b-bc62-b20c8d30413f)

**Comment by uzernamee** [comment]
Same problem.
RTX 3090, Ryzen 5900x on a x570 motherboard.
Fedora 40 installed correctly.

**Comment by nicknamenamenick** [comment]
I would recommend following the "ISO Workarounds / Alternative Installation Methods" section in the [installation guide](https://universal-blue.discourse.group/docs?topic=1146) and try method 1.  Use the Fedora Kinoite (KDE Plasma) or Fedora Silverblue (GNOME) ISO and rebase.

**Comment by AwkwaBear** [comment]
i just had this issue as well and it was due to me attempting to reinstall without properly deleting the EFI entries for the previous installation. 

I had just installed a dual boot of bazzite with windows but got stuck at a black screen. I was able to access the terminal but unable to login with my username and pw set during the install.

After trying to reinstall i got the exact error shown above.

I was able to fix it by booting into windows and using the `diskpart` tool to delete the EFI boot entry for fedora on my boot partition instructed in the following post

https://askubuntu.com/questions/429610/uninstall-grub-and-use-windows-bootloader

**Comment by Daxiongmao87** [comment]
> i just had this issue as well and it was due to me attempting to reinstall without properly deleting the EFI entries for the previous installation.
> 
> I had just installed a dual boot of bazzite with windows but got stuck at a black screen. I was able to access the terminal but unable to login with my username and pw set during the install.
> 
> After trying to reinstall i got the exact error shown above.
> 
> I was able to fix it by booting into windows and using the `diskpart` tool to delete the EFI boot entry for fedora on my boot partition instructed in the following post
> 
> https://askubuntu.com/questions/429610/uninstall-grub-and-use-windows-bootloader

What would be the solution for those who did not dual boot and have wiped our disk of Windows? I tried using fdisk/diskpart to completely wipe my drive, still no dice.

**Comment by liaralabs** [comment]
You can switch to another terminal when the install starts and use `efibootmgr` to help with this.

List boot entries with
```
efibootmgr
```

Then remove the offending entry with 
```
efibootmgr -b <bootnum> -B
```

If you are re-using an efi system partition, might be prudent to clear out things related to Fedora as well. Take care when cleaning up. When in doubt, backup/rename instead of delete.

**Comment by alexeiz** [comment]
I have no "offending" entries in the EFI boot list and still get this error.

**Comment by shishimaru90** [comment]
> I have no "offending" entries in the EFI boot list and still get this error.

Same here.

Honestly terrible.

**Comment by CossmicMussel** [comment]
Hello fellow nerds,

Believe I found out how to fix this issue at least for my scenario, it was an issue with EFI boot stuff, however the command the installer did not work.

Answer: 
My main operating system is windows so I used this app called easyUEFI on windows to completely rebuild my EFI boot, once I did this I was able to install with no issues. I’m sure there’s equivalents  to easyUEFI on other OS’s.

**Comment by liaralabs** [comment]
If you aren't dual booting, you can try wiping the ESP and letting the installer create it. Otherwise, mount the ESP and look for 'fedora' directories and delete them (assuming you aren't trying to dual boot with an existing fedora installation)

**Comment by shishimaru90** [comment]
> If you aren't dual booting, you can try wiping the ESP and letting the installer create it. Otherwise, mount the ESP and look for 'fedora' directories and delete them (assuming you aren't trying to dual boot with an existing fedora installation)

This doesn't help. I literally have a brand new UEFI partition on the disk, formatted through Bazzite's installer. 

Actually Fedora Atomic KDE has the same issue, so they might just "hate" my laptop.

**Comment by Saschabrix** [comment]
I have the same issue. 
With my AYANEO Next device. 

Main NVME is populated with an 2TB Nvme.
Installing Bazzite on an external SSD. 

I tried twice, it failed twice. 

Will wait a couple of moth more and try again.

**Comment by jnfDev** [comment]
> i just had this issue as well and it was due to me attempting to reinstall without properly deleting the EFI entries for the previous installation.
> 
> I had just installed a dual boot of bazzite with windows but got stuck at a black screen. I was able to access the terminal but unable to login with my username and pw set during the install.
> 
> After trying to reinstall i got the exact error shown above.
> 
> I was able to fix it by booting into windows and using the `diskpart` tool to delete the EFI boot entry for fedora on my boot partition instructed in the following post
> 
> https://askubuntu.com/questions/429610/uninstall-grub-and-use-windows-bootloader

This worked for me thanks

**Comment by Zeglius** [comment]
> The new based on 41 failed too.
> 
> - I completely deleted the UEFI partition
> - I created a new one and set mount point to /boot/efi (is this correct?)
> - format'd and set as / another btrfs partition
> 
> And still doesn't work.
> 
> Any chance to get a proper offline iso? I even tried with wifi 5ghz, with ethernet cable... I don't know what to try anymore to be honest. It's definitely not a safe installation method since it can fail so easily, format your disk, and find yourself without a working system. 

The iso is already offline, no internet access is required other than for downloading the iso itself.

**Comment by marcdraco** [comment]
Happened to me on a Lenovo Legion.

Good news is the EFI fix worked for me.

Bad news is that (although Windows was still there) I got into a mess trying to install the MDK do-dads for the Shim (I assume) and now .... NO Microsoft Windows. Presumably this means fixing it elsewhere but that was the most painful install I've done since the SLACKWARE days. This is poor when compared to many (most?) "popular" distros these days.

I'm very surprised at Fedora to be honest but at least we're not paying for it, so there's that.

**Comment by shishimaru90** [comment]
I had to create a EFI partition that lives only for Bazzite (or Bluefin or Aurora). Even if there's already a partition on the disk, I had to create another one. Fedora really hates to see multiboot computers.

**Comment by marcdraco** [comment]
> My penny's worth.
> 
> Really new to Linux, My Laptop is a bog standard Intel board from China. Everything thus far from the great many Distros have installed on it. I'm installing from an ISO downloaded direct from the website, where I also get ..
> 
NOTE: blank disk install, no update or duel boot. Flustered, really wanted this to work.

Oh man I felt that. I managed to wipe my Windows install (which isn't necessarily a bad thing) and I've been using Linux for decades now. I guess we're mostly used to fairly robust installers that don't mess around like this thing does. I love the idea of an atomic desktop but Clear Linux seems to do that without this carry on... 

In my case it was the EFI partition - not that the installer seems to give us any clue at all. You'll need 2-300 Mb for the EFI rubbish but try doing that manually in Blivet and reserving the remaining disk space for BTRFS. I had to turn off secure boot and got myself  

I think Blivet is the name of the partitioner, I'm a Debian12  guy normally - and  mean Debian not Ubuntu. I haven't used Fedora much in ... years. And here we are - at least I finally got mine working. I only wanted to run my Steam Deck games without the hassle and... that failed spectacularly so far. Which means I'm having to reinstall stuff until I have time to figure out how to mount the same physical library on multiple machines.

**Comment by antonio0831** [comment]
Same issue. The EFI fix didn't work for me

**Comment by shishimaru90** [comment]
This is what worked for me:
- on the same disk, no shared systems (only works if OSs are on different disks) 
- this means that it must have its own EFI partition. Don't share it.
- I don't know why, but if I prepare the partitions in advance and/or if I format them and use mount points, the installation fails...
- ...so I resolved like this: erased all the partitions, recreated them in Anaconda AND also added a /boot partition (I used ext2). If I don't create a /boot partition, it just doesn't work, I don't know why.


Now you'd ask why I just didn't use the automatic installation. It's because I have a big partition with data on it and I need it on that partition, so I had to go with the manual setup.

**Comment by Blockzilla101** [comment]
I'm having the same problem, the suggestions from previous commenters did not work for me. (at least ones I can try without losing data)

For context, I wanted to switch from arch to this and have a dual boot setup. I used manual partitioning & booted the iso using ventoy (incase that has something to do with it). 

I configured the partitions to basically what I had previously but only making the installer overwrite my root partition. Didn't work. Tried again with the efi partition nuked & recreated it to 500mb, still didn't work.

- Partition Setup:
  - 2 NTFS partitions (windows + windows data)
  - `ext4 /home`
  - ` ext4 / `
  - `fat32 /boot/efi`

I cannot switch to btrfs as the guide suggested as I do not have a spare drive to backup my data. I also didn't know about their partition suggestion until after I gave up.

![Image](https://github.com/user-attachments/assets/e41386b4-4024-4bf1-b141-734aa962f414)

**Comment by drstrider** [comment]
> I would recommend following the "ISO Workarounds / Alternative Installation Methods" section in the [installation guide](https://universal-blue.discourse.group/docs?topic=1146) and try method 1. Use the Fedora Kinoite (KDE Plasma) or Fedora Silverblue (GNOME) ISO and rebase.

It looks like that page has been deleted, does anyone know what it said? I'm blocked by this bug too 😔

**Comment by pieterlexis** [comment]
I see a similar problem with the latest iso. I see these errors in the logs:

![Image](https://github.com/user-attachments/assets/fc63f72c-d064-470e-8e3b-e419ce65ed85)
![Image](https://github.com/user-attachments/assets/8963882e-0d4f-440c-a05c-643a77b35cca)

The error I see happens in the post-install step when the `ostree admin instutil set-kargs rhgb quiet root=UUID=a0e6fbbc-e44c-494c-b1cc-1faacc386b57 rootflags=subvol=@bazzite rw` command is run. This outputs the following error:

```
Unable to find a deployment in sysroot
```

**Comment by LaercioSantana** [comment]
I was with the same error here. I found that atomic images need a separated partition for /boot.

So we need adds a minimum of 3 mount points to make it works: `/boot/efi`, `/boot` and `/`

What I did:

/boot           ext4                  2gb
/boot/efi      EFI system        2gb
/                   Btrfs                 80gb
/home          ext4                 100gb

For reference:
https://discussion.fedoraproject.org/t/fedora-kinoite-40-install-always-fail-when-installing-bootloader/116688/10
https://github.com/fedora-silverblue/issue-tracker/issues/284#issuecomment-2002593455

**Comment by nefarius** [comment]
Same issue I ran into. I had the `/boot/efi` partition on NVME 1 (where Windows was before) and my Linux install on NVME 0, so I migrated the EFI from NVME 1 to NVME 0, completely wiped NVME 1 and put Bazzite on it and then it installed without issues.

**Comment by atimeofday** [comment]
> So we need adds a minimum of 3 mount points to make it works: `/boot/efi`, `/boot` and `/`
> 
> What I did:
> 
> /boot ext4 2gb 
> /boot/efi EFI system 2gb 
> / Btrfs 80gb 
> /home ext4 100gb

I am missing context here so I hope this is relevant, but I saw something I believe I can correct and point to a useful resource for. Here is documentation for dual-booting Bazzite which includes the correct partitioning scheme for Fedora Atomic images:

https://docs.bazzite.gg/General/Installation_Guide/dual_boot_setup_guide/

```
Manual Partitioning Scheme:

    mount point: /boot/efi
    format:      EFI system partition
    size:        300MB

    mount point: /boot
    format:      ext4
    size:        1GB

    mount point:
    format: btrfs
    size: [max]

    mount point: /
    format:      btrfs (subvolume)

    mount point: /var
    format:      btrfs (subvolume)

    mount point: /var/home
    format:      btrfs (subvolume)
```

**Comment by HaedHutner** [comment]
I've had the same problem and I've been trying to fix it for the last 3 or 4 weeks ( off and on project )

![Image](https://github.com/user-attachments/assets/905a1041-0869-4c2f-9589-b171a4fc0e67)

![Image](https://github.com/user-attachments/assets/8cd31c40-7aef-4551-8e99-21b95cb25a68)

As can be seen from the images, no EFI boot entries other than the thumbdrive I used to launch a debian/gparted from.

I'm not trying to dual-boot. All I want is to just plainly install bazzite and nothing else, on a blank NVME drive, and I also have 2 SSDs for storage hooked up as well.

**Comment by birdlavv** [comment]
I ran into this issue on a disk that had Windows previously.

This is what solved the issue for me:

1. Device selection
2. YES - Automatic
3. YES - Free up space by removing or shrinking existing partitions
4. Pick the drive to nuke
5. Hit done
6. (May have to do 1-5 twice to trigger popup...?)
7. You will get a popup
8. Hit "Delete all"
9. Hit "Reclaim space"
10. Begin installation

**Comment by Ttxman** [comment]
I had same issue yesterday, while trying to install & dualboot bazzite 41 on windows disk.
What helped me in the end was manually deleting `/EFI/Fedora` directory on the EFI partition.

**Comment by dedors** [comment]
I run into this issue on my first install attempt, on the disk where windows is. Now I can't even boot with the USB stick anymore.
I followed the installation and the dual boot instructions exactly. 
I leave this comment to show there are users that give up when in such a situation. I had the trouble of choosing a distro, chosen this one and it feels like not worth to start troubleshooting that early into a new OS. That's why I will move on.

**Comment by atimeofday** [comment]
@JSblah Bazzite uses the official Fedora installer. There is ongoing work by Fedora and Red Hat to replace it, but it is a years-long project. I do not recognize the actions you describe from the dual-boot instructions I helped write. There should be no way to interfere with the UEFI itself unless you manually do something unnecessary and extraordinarily wrong.

Since every new comment on this issue is getting dramatically further away from the issue topic instead of reproducing it, it may be time to close it.

**Comment by Ttxman** [comment]
It happened again, and I unstucked it same way as last time by removing everything non windows by `efibootmgr` and then deleting `/EFI/Fedora` folders on boot partition (from windows). 
My current guess is that 100MB EFI boot partition (created by windows years ago) is too small for whatever is the installer trying to do.

Dualboot with windows 10

- I had Bazzite 41 installed on the same disk (in separate btrfs partition) as windows 
- Launched the Bazzite 41 installer again with the intent of "clean install"
- changed the keyboard layout and renamed network adapters
- removed all created linux partitions from the installer and let them be automatically recreated (in the simple interface, the partition deletion does not show the name of the disk in confirmation dialog....)
- shrinked btrfs partition and added swap partition (I have issues with hibernation and just messing with things)
- installation failed, 
- and subsequent attempt to reinstall on the existing partition failed too.

**Comment by atimeofday** [comment]
@Ttxman 

> My current guess is that 100MB EFI boot partition (created by windows years ago) is too small for whatever is the installer trying to do.

This is probably true. My (2020) Windows EFI partition is 273MB, and my separate Bazzite EFI partition is 262MB.

> Dualboot with windows 10
> * I had Bazzite 41 installed on the same disk (in separate btrfs partition) as windows

Not recommended or supported but possible and understandable if necessary. The comment "in separate btrfs partition" tells me you may not understand enough about partitions and probably did not correctly set up the 5 partitions (technically 3 partitions and 3 subvolumes) necessary for Bazzite to function.

> * Launched the Bazzite 41 installer again with the intent of "clean install"
> * changed the keyboard layout and renamed network adapters
> * removed all created linux partitions from the installer and let them be automatically recreated (in the simple interface, the partition deletion does not show the name of the disk in confirmation dialog....)

All unnecessary partitions should be deleted and removed before beginning the installation process. Automatic partition creation is generally recommended but does not always work, and is on an entirely separate path from some of the actions you describe doing, so you may have reverted changes or failed to set up any partitions by juggling parts of the installer.

> * shrinked btrfs partition and added swap partition (I have issues with hibernation and just messing with things)

This is not part of the installation process. Fedora and Bazzite avoid using a swap partition for a reason. You may be able to add it later, but your issue report is useless with so many unsanctioned steps which may cause arbitrary failures.

> * installation failed,
> * and subsequent attempt to reinstall on the existing partition failed too.

Yeah, that makes sense. You should not be installing on existing copies of any of the five necessary partitions, and attempting to install Bazzite (or any Fedora Atomic distro) on less than five partitions will always fail. Please refer back to the dual booting instructions I linked and described in a prior comment, and follow them precisely. If that works, then you can perform additional modifications such as introducing a swap partition.

**Comment by Ttxman** [comment]
@atimeofday I don't need assistance. As I wrote before I have working Bazzite system (created with exact same steps with the swap for hibernation). 
I've just hoped, that my guess about low free space (or files already existing there) in preexisting `/BOOT/EFI` may be common issue for some people in this thread. (which became active again in last few days)

Right now manualy deleting the /EFI/Fedora files and all fedora boot records allow me to do single Bazzite installation. All subsequent installations fails regardless how I delete the partitions (I tried diskpart from windows and both tools in the installer) and regardless if I let the installer fill the free disk space automatically or I create the partitions manually.
I may try to recreate/resize the EFI partition in the ongoing weeks (and report back), but I need to have a free day for backups and to repair the damage after I inevitably mess it up :)

**Comment by atimeofday** [comment]
@Ttxman Ahhh, I see. In that case, I concur that reusing a (too-small) existing EFI partition may be causing issues for people. It's not the only cause, though - one of the shortfalls of the current (read: legacy, not yet replaced) Fedora installer is that it tends to provide the same unhelpful error message for a large variety of different failure cases. The only consistent way to get past the unknown causes of the error message is just to triple down on correctly and precisely following the given installation instructions.

A second EFI partition is the way to go. The second copy of Grub can discover and add the Windows bootloader, and it ***might*** be able to prevent Windows from nuking Grub by resetting its EFI partition on some random update. The downside of making a second EFI partition is that for certain multi-OS configurations it becomes necessary to go through the BIOS/UEFI boot menu to switch between OSes which one or more versions of Grub cannot discover/add.

Backups are good. It took me many tries to iron out how to install / dual boot Silverblue and then Bazzite before I was able to write the first few drafts of the dual booting guide. When I started out, I had to reverse-engineer the manual partitioning scheme from someone's fstab file because there just wasn't really any documentation to manually install or dual boot Fedora Atomic images.

**Comment by JSblah** [comment]
Providing a larger EFI partition fixed the issue for me. 

> [@JSblah](https://github.com/JSblah) Bazzite uses the official Fedora installer, because it literally is Fedora. There is ongoing work by Fedora and Red Hat to replace it, but it is a years-long project. I do not recognize the actions you describe from the dual-boot instructions I helped write. There should be no way to interfere with the UEFI itself unless you manually do something unnecessary and extraordinarily wrong.

It seems the UEFI stub for Fedora was not playing nicely with my existing firmware configuration. I don't think it was critical but it definitely freaked me out. I decided to give this another shot and had a successful install. It seems the problem was the existing EFI was too small. I did everything else the same as before but this time I wiped my drive before reinstalling MacOS I laid out a custom sized 800MB EFI. After installing MacOS and shrinking the APFS I booted anaconda and created the boot partition and btrfs container volumes and the install went smoothly. It was a huge PITA but in the end mission accomplished and I couldn't have done it without the dual boot guide so thanks for your work guys.

**Comment by atimeofday** [comment]
> I did everything else the same as before but this time I wiped my drive and reinstalled MacOS with a custom sized 800MB EFI. After installing MacOS and shrinking the APFS I booted anaconda and created the boot partition and btrfs container volumes and the install went smoothly. It was a huge PITA but in the end mission accomplished and I couldn't have done it without the dual boot guide so thanks for your work guys. 

Progress! Glad to hear at least the installation itself worked. Preaching to the choir about it being a pain; I couldn't count how many times I went through that manual partitioning scheme to map out how to do it. 

> Nope, after installing Fedora my BIOS menu still freezes when I try and access the boot menu. Never experienced that before. I will see if a BIOS update will fix it and may be opening another issue... 

BIOS update sounds good. I'll try to remember to keep an eye out for another issue in case it doesn't work. Worst case, the last thing I can think of right now to try would be making a fully separate second EFI partition for Bazzite. It looks like it shouldn't work, but it can, and it makes some extra things possible.

**Comment by solarisfire** [comment]
Completely brand new drive, fresh out of the box, and getting this:

![Image](https://github.com/user-attachments/assets/3ebf4de7-fe85-4f55-bd3d-9f644663699c)

Not a great first impression of the OS.

**Comment by atimeofday** [comment]
> Completely brand new drive, fresh out of the box, and getting this:
> 
> ![Image](https://github.com/user-attachments/assets/3ebf4de7-fe85-4f55-bd3d-9f644663699c)
> 
> Not a great first impression of the OS.

The maintainers are eager to some day replace the upstream legacy Fedora installer which causes this issue. The solution to that error as it currently stands is often to just try again.

**Comment by JSblah** [comment]
> Completely brand new drive, fresh out of the box, and getting this:
> Not a great first impression of the OS.

This is not the OS but the extremely picky Fedora installer. Did you do manual partitioning or something else? The issue seems to be that the installer wants a specific disk layout. Your disk should be set up as GPT and the partition map needed is outlined above somewhere and needs to be like this:

    mount point: /boot/efi
    format:      EFI system partition
    size:        300MB

    mount point: /boot
    format:      ext4
    size:        1GB

    mount point:
    format: btrfs
    size: [max]

    mount point: /
    format:      btrfs (subvolume)

    mount point: /var
    format:      btrfs (subvolume)

    mount point: /var/home
    format:      btrfs (subvolume)

The subvolume mount points are created differently than the other partitions and are part of the btrfs container. A couple of people (including me) had the error because of the EFI partition. It needs to be 300MB or larger and have the proper mount point. Good luck.

**Comment by marcdraco** [comment]
Same for me - just a case of try again until it works. And when it does, it's very good. The only issue I have (I don't want to open a new issue as it's likely already under investigation) is I can't update Proton inside the OS. If anyone knows a fix for that I'd love to have it as I can't get stuff like DCS to run and the updater is constantly reminding me to update.

**Comment by solarisfire** [comment]
> > Completely brand new drive, fresh out of the box, and getting this:
> > Not a great first impression of the OS.
> 
> This is not the OS but the extremely picky Fedora installer. Did you do manual partitioning or something else? The issue seems to be that the installer wants a specific disk layout. Your disk should be set up as GPT and the partition map needed is outlined above somewhere and needs to be like this:
> 
> ```
> mount point: /boot/efi
> format:      EFI system partition
> size:        300MB
> 
> mount point: /boot
> format:      ext4
> size:        1GB
> 
> mount point:
> format: btrfs
> size: [max]
> 
> mount point: /
> format:      btrfs (subvolume)
> 
> mount point: /var
> format:      btrfs (subvolume)
> 
> mount point: /var/home
> format:      btrfs (subvolume)
> ```
> 
> The subvolume mount points are created differently than the other partitions and are part of the btrfs container. A couple of people (including me) had the error because of the EFI partition. It needs to be 300MiB or larger and have the proper mount point. Good luck.
> 
> ...it just occurred to me that this could be a case of distinguishing between MB and MiB where 300MB is only 286MiB. Not sure which unit the Fedora installer uses but setting the EFI size to 316 (either unit) might solve.

It was a brand new disk, I told the installer to take the whole thing and do what it wants with it...

I've never seen a Linux install fail after doing that before.

**Comment by marcdraco** [comment]
I've had a few installs fail on Linux - usually more esoteric ones - but Bazzite gave me some real headaches. When it works, it works very well indeed but I'd probably stick to one of the alternatives if I have re-install the Steam Deck again - largely because of the inability to update proton. I just don't have the energy to do it all again when it's working.

**Comment by solarisfire** [comment]
I've been using Linux for over 20 years... I've used Fedora since Fedora Core 1 installed from the CD glued to the front copy of Linux Format the month it was released.

I've used some very weird and wonderful distros over the years.

I have never had a Linux OS not create the correct partition map for me when telling it to set things up automatically before.

**Comment by marcdraco** [comment]
I think the issue with the installer is the fall-back to a working system in case something breaks. I don't know how or if that's baked into Fedora (I'm a Debian guy). But there are several other distros that are less painful to install like Chimera but they don't have anywhere near as much protection if something goes south. So far (and I'm tempting fate) Bazzite has been rock solid - AFTER I got it working.

**Comment by noelmiller** [comment]
This issue has been plaguing us since the creation of the ISOs. We are aware of this issue and are actively working on a replacement effort for the existing install process. This is unfortunately an upstream issue with Anaconda and until we have a replacement ready, there is not a lot we can do.

**Comment by mailnike** [comment]
Same issue on the GPD device with the existing Windows OS. :-( I hope there will be a solution soon.

**Comment by alshdavid** [comment]
Same issue with me using KDE Plasma and ROG Ally (first gen). I have Windows installed and am trying to dual-boot with Bazzite.

First attempt: failed
I have 500gb free unpartition space and used auto partitioning. Windows is already installed in the other partitions.

Second attempt: failed
I manually created the following partitions into the free space (Windows was still installed)
|Mount point|Format|
|-|-|
|`/boot/efi`|EFI Partition|
|`/`|btrfs|

Third attempt: Install only Bazzite (btrfs) : failed
Fourth attempt: Install only Bazzite (ext4) : failed

🤷

**Comment by aaravrav** [comment]
I had a small pre-existing EFI partition. Making and selecting a new EFI partition with the required size fixed the issue.

**Comment by alshdavid** [comment]
Same. Having this layout works for me. It also works if this layout is sitting next to the Windows partitions

|Mount point|Format|Size|
|-|-|-|
|`/boot/efi`|EFI Partition|400mb|
|`/boot`|ext4|1gb|
|`/`|btrfs|*gb|

**Comment by connertron** [comment]
Enable secureboot then run installer again should work.

**Comment by Diavelaaron** [comment]
I think mine was solved by making the following two changes to the boot disk using Rufus.
1) I changed the USB bootdisk from MBR to GPT
2) I changed to DD image type (after Rufus warns you about hybrid ISO image).

This seemed to fix the installer for me.

**Comment by Xarianne** [comment]
I just had this problem. I installed Bazzite with prior EFI partitions before so this blindsided me.

The only thing I can think of is that this time I also had another partition for my data on the install disk, which I chose to preserve, rather than deciding to format the entire disk. So maybe that caused the previous EFI partition to be a problem, maybe the formatting happens at a different stage if you don't clean out the entire disk?

My Windows decided to blue screen on me when I tried to delete it, so I wound up booting up via my trusty Linux Mint rescue stick (I only use it for this kind of stuff I don't actually use Mint). Got it to delete my EFI partition and then the install worked.

**Comment by richo** [comment]
I'm also getting this error message, reading through the replies and digging through the advanced partitioner it appeared that the installer was trying to use the smaller EFI partition windows had created, and not the 300mb one that Bazzite was creating.

I took the plunge and erased my windows partitions, but it still fails with this error.

**Comment by marcdraco1-gif** [comment]
The only way I found past this bug is to use the partitioner (and you may find it easier to use a different distro like Debian) to remove the whole partition map and start over. If you start with a blank partition map (like a blank drive) it seems to work OK.

**Comment by dracinn** [comment]
This is just a wild guess as I know nothing about Linux in general.

I wonder how much RAM do you guys have as this maybe is an issue with the live image.

**Comment by richo** [comment]
In my instance it turned out the install media I created was physically defective.

**Comment by dracinn** [comment]
> In my instance it turned out the install media I created was physically defective.

Is there a hash value I can check?

**Comment by dracinn** [comment]
I think it might be easier to get a Universal Blue image and just rebaseing instead.

**Comment by Xarianne** [comment]
> This is just a wild guess as I know nothing about Linux in general.
> 
> I wonder how much RAM do you guys have as this maybe is an issue with the live image.

I have 32 GB of RAM and I only have issues with custom ISOs. An issue was opened upstream and it's known: https://github.com/JasonN3/build-container-installer/issues/150

**Comment by alshdavid** [comment]
Running into this issue again. I tried manually partitioning but I think the Bazzite installer's manual partitioner has been simplified? I can't figure out how to use it.

**Comment by alshdavid** [comment]
Ok, first I disabled secure boot, then before running the Bazzite installer I used GParted to set up the following partition layout in GPT format;

|Mount point|Format|Size|Flags|
|-|-|-|-|
|`/boot/efi`|fat32|400mb|esp, boot|
|`/boot`|ext4|1gb|
|`/`|ext4|*gb||

Then I used the installer "assign mounts" and pointed the installer to the preconfigured partitions, making sure I didn't reformat the partitions. 

You can use this same layout for dual booting, just resize the `/` partition as needed. I use the BIOS boot-device selector for selecting the OS rather than GRUB (I find it less fiddly).
